cnn_framework/predictor_test.py

```diff
@@ -49,18 +49,14 @@ def test_load_input(self):
         dense2 = self.predictor.model.layers[3]
 
         self.assertEqual(gfp.W_inner.shape.eval()[0], 4)
-        #self.assertEqual(gfp.W_inner.shape.eval()[1], 38)
-        #self.assertEqual(gfp.W_inner.shape.eval()[2], 38)
-        self.assertEqual(gfp.W_inner.shape.eval()[1], 37)
-        self.assertEqual(gfp.W_inner.shape.eval()[2], 37)
+        self.assertEqual(gfp.W_inner.shape.eval()[1], 38)
```

Comment:
Ah ok, I hadn't gotten to this commit yet in my review, so that probably answers my question about whether I can load already-trained weights. Also, please squash this with the commit where you introduce the changes reverted here (or just drop both of them).

Reply:
Done